datapackage/resource_new.py

```diff
@@ -0,0 +1,267 @@
+from __future__ import absolute_import
```

Comment:
I really don't like `*_new` file names, as they eventually stop being new. Is there a reason why you need to keep the current `resource.py` and not replace it? If there is can you call the old one `resource_legacy.py` and this one `resource.py`?